jsk_arc2017_baxter/euslisp/pick-main.l

```diff
@@ -185,20 +191,23 @@
           (send *ri* :spin-off-by-wrist 5)
           (send *ri* :wait-interpolation)
           (ros::ros-info "[main] ~a, return object in shelf" arm)
+          (send *ri* :add-shelf-scene)
           (send *ri* :fold-pose-back arm)
           (send *ri* :wait-interpolation)
           (setq state :set_target)
           (send *ri* :set-arm-state-param arm state))
         (:place_object
           (ros::ros-info "[main] ~a, ~a" arm state)
           (ros::ros-info "[main] ~a, place object in bin ~a" arm target-cardboard)
+          (send *ri* :add-shelf-scene)
           (send *baxter* :head_pan :joint-angle (if (eq arm :larm) 80 -80))
           (send *ri* :angle-vector
                 (send *ri* :ik->cardboard-entrance arm target-cardboard
                       :offset #f(0 0 300) :rotation-axis :z :use-gripper t)
                 :fast (get-arm-controller arm) 0 :scale 5.0)
           (send *ri* :wait-interpolation)
-          (send *ri* :angle-vector (send *baxter* arm :move-end-pos #f(0 0 -300) :world)
+          (send *ri* :delete-cardboard-scene arm)
+          (send *ri* :angle-vector-raw (send *baxter* arm :move-end-pos #f(0 0 -300) :world)
```

Comment:
Why do you use `:angle-vector-raw`? You delete cardbard scene so that you can move robot in cardboard, don't you?